l5kit/l5kit/data/labels.py

```diff
@@ -17,4 +17,12 @@
     "PERCEPTION_LABEL_ANIMAL",
     "AVRESEARCH_LABEL_DONTCARE",
 ]
-LABEL_TO_INDEX = {label: index for (index, label) in enumerate(LABELS)}
+PERCEPTION_LABEL_TO_INDEX = {label: index for (index, label) in enumerate(PERCEPTION_LABELS)}
+
+
+Tl_FACE_LABELS = [
+    "ACTIVE",
+    "INACTIVE",
+    "UNKNOWN",
+]
+Tl_FACE_LABEL_TO_INDEX = {label: index for (index, label) in enumerate(Tl_FACE_LABELS)}
```

Comment:
Nit: could use an enum, rather than use list and get the label index

Reply:
you're right we can probably convert a lot of this strings literals we use for indexing to enum